pkgs/applications/editors/emacs/elisp-packages/melpa-packages.nix

```diff
@@ -718,6 +733,812 @@ let
+
+        projmake-mode = ignoreCompilationError super.projmake-mode; # elisp error
+
+        # https://github.com/tumashu/pyim-basedict/issues/4
+        pyim-basedict = addPackageRequires super.pyim-basedict [ self.pyim ];
+
+        # TODO report to upstream
+        realgud-lldb = super.realgud-lldb.overrideAttrs (old: {
+          preBuild =
+            old.preBuild or ""
+            + "\n"
+            + ''
+              rm --verbose cask-install.el
+            '';
+        });
+
+        # empty .yas-compiled-snippets.el causing native-compiler-error-empty-byte
+        requirejs = ignoreCompilationError super.requirejs;
+
+        rhtml-mode = ignoreCompilationError super.rhtml-mode; # elisp error
+
+        roguel-ike = ignoreCompilationError super.roguel-ike; # elisp error
+
+        rpm-spec-mode = ignoreCompilationError super.rpm-spec-mode; # elisp error
+
+        # https://github.com/emacsfodder/emacs-theme-sakura/issues/1
+        sakura-theme = addPackageRequiresIfOlder super.sakura-theme [ self.autothemer ] "20240921.1028";
+
+        scad-preview = ignoreCompilationError super.scad-preview; # elisp error
+
+        # https://github.com/wanderlust/semi/pull/29
+        # missing optional dependencies
+        semi = addPackageRequires super.semi [ self.bbdb-vcard ];
+
+        shadchen = ignoreCompilationError super.shadchen; # elisp error
+
+        # missing optional dependencies and one of them (mew) is not on any ELPA
+        shimbun = ignoreCompilationError (
+          addPackageRequires super.shimbun [
+            self.apel
+            self.flim
+            self.w3m
+          ]
+        );
+
+        slack = mkHome super.slack;
+
+        # https://github.com/ffevotte/slurm.el/issues/14
+        slurm-mode = addPackageRequires super.slurm-mode [
+          self.dash
+          self.s
+        ];
+
+        smart-tabs-mode = ignoreCompilationError super.smart-tabs-mode; # elisp error
+
+        # needs network during compilation
+        # https://github.com/md-arif-shaikh/soccer/issues/14
+        soccer = ignoreCompilationError (addPackageRequires super.soccer [ self.s ]);
+
+        # elisp error and missing optional dependencies
+        soundklaus = ignoreCompilationError super.soundklaus;
+
+        # missing optional dependencies
+        sparql-mode = addPackageRequires super.sparql-mode [ self.company ];
+
+        speechd-el = ignoreCompilationError super.speechd-el; # elisp error
+
+        spu = ignoreCompilationError super.spu; # elisp error
+
+        # missing optional dependencies
+        ssh-tunnels = addPackageRequires super.ssh-tunnels [ self.helm ];
+
+        # https://github.com/brianc/jade-mode/issues/73
+        stylus-mode = addPackageRequires super.stylus-mode [ self.sws-mode ];
+
+        # missing optional dependencies
+        suggest = addPackageRequires super.suggest [ self.shut-up ];
+
+        symex = ignoreCompilationError super.symex; # elisp error
```

Comment:
I[ reported a bug to the symex maintainer](https://github.com/drym-org/symex.el/issues/148#issuecomment-2431312021), but it turns out that they couldn't replicate it.  We managed to fix it anyhow, but it makes me wonder if something is subtly amiss here.

Reply:
> My theory is that the underlying error--which this ignores--was recently fixed.

I do not think so.  Many errors reported in https://github.com/drym-org/symex.el/issues/130 are in the build [log].  Those errors are not specific to nix and should be fixed upstream. 

[log]: https://hydra.nixos.org/build/273770270/nixlog/1